### DISCUSSION TITLE
Enable sorting of cards in ShowHabitActivity

### DISCUSSION THREAD

**Discussion by chennemann** [iSoron/uhabits#549] [Feature Requests]
I think it would be great to enable the user to sort the widgets in the habit details view. That would make problems like the following obsolete:

1. I think the best place to show it is as a new card in the ShowHabitActivity, either directly below the current SubtitleCard, or after the OverviewCard. (Originally posted by @iSoron in https://github.com/iSoron/uhabits/pull/547#issuecomment-572012252_)

It would also enable the user to sort the information according to his preferences. For some the calendar is more important than the Score graph. Some others might want to see the Best streaks or the history immediately.

**Comment by iSoron** [comment]
I agree this would be useful. Pull request are welcome. We should also figure out how to make this feature discoverable.

**Comment by me-manikanta** [comment]
A quick question if we are going to implement this feature:
Will this be a global setting by that I mean will this be applicable to all habits or whether this is habit specific?

If this is habit specific then the only way to show this to the user is in the menu button of the ShowHabitActivity, if not then we can actually show this in all settings menu

I hope I am clear and not adding more confusions here
